src/tests/unit/tests/injected/tab-stops-requirement-evaluator.test.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Tests look good but should be broken by case to be consistent our usual test style.